packages/connect/src/backend/Blockchain.ts

```diff
@@ -120,7 +120,9 @@ export class Blockchain {
 
         // There is no `rippled` setting that defines which network it uses neither mainnet or testnet
         // see: https://xrpl.org/parallel-networks.html
-        const shortcut = this.coinInfo.shortcut === 'tXRP' ? 'XRP' : this.coinInfo.shortcut;
+        let shortcut = this.coinInfo.shortcut === 'tXRP' ? 'XRP' : this.coinInfo.shortcut;
+
+        shortcut = shortcut.toLowerCase() === 'bnb' ? 'bsc' : shortcut;
```

Comment:
don't we control the shortcut? why is it?

Reply:
Blockbook uses shortcut `BNB` because `bnb` is native currency, but we try to solve it